src/Util/AssetHookResolver.php

```diff
@@ -41,15 +41,20 @@ public function resolve(): array
     {
         $isLogin = $this->context->isLogin();
         $isFront = $this->context->isFrontoffice();
+        $isActivate = $this->context->isWpActivate();
```

Comment:
This is fine, but I think also [`lastHook()`](https://github.com/inpsyde/assets/blob/34_asset_activate/src/Util/AssetHookResolver.php#L78) should support `isWpActivate()`.

Reply:
yep, makes sense. Added here: https://github.com/inpsyde/assets/commit/7725970eea031d70bc877ea8029225e82f116974